MAINTAINERS.md

```diff
@@ -0,0 +1,21 @@
+The current maintainers of this repo are [@adamralph](https://github.com/adamralph), [@bording](https://github.com/bording), and [@WojcikMike](https://github.com/WojcikMike).
+
+The maintainers [watch](https://github.com/Particular/NServiceBus.RabbitMQ/watchers) this repo and undertake the following responsibilities:
```

Comment:
I'm not 100% sure but I think you may be right. I googled around for some examples and use of the base verb form does indeed seem to be more common in such lists. I'll change.